<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备操作与维护软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>寄存器:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterAddrButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>地址:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>21</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterDataButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>数据:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RegisterReadCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>读</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RegisterWriteCheckBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>写</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RegisterAddrLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RegisterDataLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EepromButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>EEPROM:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EepromDataLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EepromDataButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>数据:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EepromReadCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>读</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EepromAddrLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EepromAddrButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>地址:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EepromWriteCheckBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>写</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentTemperatureButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>当前温度:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrnetTemperatureLineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentVoltageLineEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentVoltageButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>当前电压:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SoftWareUpdateButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>软件更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeviceStatButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设备状态:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StatButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>查询中...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendGprsMessageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送短信:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZigbeeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>zigbee:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TelephoneNumberButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>手机号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TelephoneNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MessageLineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MessageButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>信息:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZigbeeMessageButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>信息:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZigbeeMessageLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>431</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BeepButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>报警器:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BeepOpenCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BeepCloseCheckBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LedButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LED:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LedCloseCheckBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 关闭</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LedOpenCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LedFlashTimeLineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LedFlashTimeButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>闪烁周期:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HardWareTestSelf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设备自检:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HardWareTestSelfLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>521</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="CpuFreqLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CpuFreqButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>cpu频率:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FreeMemButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余内存:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FreeMemLineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FreeDiskSpaceButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余空间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FreeDiskSpaceLineEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>801</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="menu">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>-4</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;设备操作与维护软件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>20</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterSettingButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EepromSettingButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMessageButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendZigbeeMessageButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BeepSettingButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LedSettingButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HardWareTestSelfCheckButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
